assets/js/navbar.js

```diff
@@ -20,13 +20,22 @@ const controllerNavbar = (function (jQuery) {
       const dropdownToggle = dropdownMenu.find('.dropdown-toggle');
       const dropdownMenuContent = dropdownMenu.find('.dropdown-menu');
 
-      dropdownToggle.on('mouseenter', function () {
-        dropdownMenuContent.addClass('show');
-      })
-
-      dropdownMenu.on('mouseleave', function () {
-        dropdownMenuContent.removeClass('show');
-      })
+      // Toggle dropdown menu when clicking the dropdown toggle
+      dropdownToggle.on('click', function (e) {
+        e.preventDefault(); // Prevent default anchor behavior
+        
+        // Close all other dropdowns
+        jQuery('.dropdown-menu').not(dropdownMenuContent).removeClass('show');
```

Comment:
Please also check the line 7, because you changed the className and I think is important to add `NAVBAR_CLASS_ACTIVE` style in case it is in the page.